docker/jvm/jsa_launch

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+KAFKA_CLUSTER_ID="$(opt/kafka/bin/kafka-storage.sh random-uuid)"
+opt/kafka/bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c opt/kafka/config/kraft/server.properties
+KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=kafka.jsa" opt/kafka/bin/kafka-server-start.sh opt/kafka/config/kraft/server.properties &
+PIDS=$!
+
+sleep 10
```

Comment:
`sleep` is not the best way to wait for an application to be ready. We could check for something that indicates the app is ready (for e.g. checking if 9092 is ready to accept connections) and then maybe sleep a few seconds if required.

Reply:
Replaced static sleep with conditional loops with timeouts